bitbucket-pipelines.yml

```diff
@@ -20,7 +20,9 @@ definitions:
               AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
               CFN_ROLE: ${CFN_ROLE}
               DEBUG: ${CI_DEBUG}
-
+              UPLOAD_BADGE: true
+              APP_USERNAME: ${APP_USERNAME}
```

Comment:
What does this refer to? Can you add something to the `README.md` to explain what app this is referring to?

Reply:
I have it in the readme for the pipe, did we want to add a pipeline variables section to this repos readme as well?

Reply:
@TheOrangePuff,

Updated the readme with a little explanation.